src/lib.rs

```diff
@@ -733,6 +755,11 @@ impl<T: Clone + Integer> One for Ratio<T> {
     fn one() -> Ratio<T> {
         Ratio::new_raw(One::one(), One::one())
     }
+
+    #[inline]
+    fn is_one(&self) -> bool {
+        self.numer == self.denom
```

Comment:
Should we consider the degenerate `0/0` case?

Reply:
I mean, considering how `0/0` is an invalid state for `Ratio`, I'd assume that it shouldn't be worried about.

Reply:
That's fair, and `is_zero` doesn't worry about it either.

Reply:
Thinking about it, I'll add some `debug_assert`s to verify that the denominator is nonzero so that these can be caught in debug mode, but won't be checked on release.